docs/remote_inference_blueprints/sagemaker_connector_blueprint.md

```diff
@@ -0,0 +1,39 @@
+### Sagemaker connector blueprint example for embedding:
+
+```json
+POST /_plugins/_ml/connectors/_create
+{
+  "name": "<YOUR MODEL NAME>",
+  "description": "<YOUR MODEL DESCRIPTION>",
+  "version": "<YOUR MODEL VERSION>",
+  "protocol": "aws_sigv4",
+  "credential": {
+    "access_key": "<PLEASE ADD YOUR AWS ACCESS KEY HERE>",
+    "secret_key": "<PLEASE ADD YOUR AWS SECRET KEY HERE>",
+    "session_token": "<PLEASE ADD YOUR AWS SECURITY TOKEN HERE>"
+  },
+  "parameters": {
+    "region": "<PLEASE ADD YOUR AWS REGION TOKEN HERE>",
+    "service_name": "sagemaker"
+  },
+  "actions": [
+    {
+      "action_type": "predict",
+      "method": "POST",
+      "headers": {
+        "content-type": "application/json"
+      },
+      "url": "<PLEASE ADD YOUR Sagemaker MODEL ENDPOINT URL>",
+      "request_body": "<PLEASE ADD YOUR REQUEST BODY. Example: [\"${parameters.inputs}\"]>"
```

Comment:
How about just `"{parameters.inputs}"` ? Add `[` and `]` looks confusing as that's only for embedding generation , not a general one

Reply:
You mean: `${parameters.inputs}` ?